docs/installation/installation-using-docker-linux.md

```diff
@@ -28,33 +28,32 @@ cd project-base
 - *The `--no-install` option disables installation of the vendors - this will be done later in the Docker container.*
 - *The `--keep-vcs` option initializes GIT repository in your project folder that is needed for diff commands of the application build and keeps the GIT history of `shopsys/project-base`.*
 
-### 2.1 Use install script
-In case you want to start demo of the app as fast as possible, you can now execute install script.
-
-```
-./scripts/install.sh
-```
-
-If you want to know more about what is happening during installation, continue with next step.
-
-### 2.2 Create docker-compose.yml file
+### 2. Installation
+Now, you have two options:
+- **Option 1:** In the case you want to start demo of the application as fast as possible, you can simply execute the installation script and that is all:
```

Comment:
I'm missing an information about where to continue -> to installation-using-docker-application-setup.md see it in your browser